typedmodels/models.py

```diff
@@ -453,6 +443,24 @@ def _get_unique_checks(self, exclude=None, **kwargs):
         return unique_checks, date_checks
 
 
+def _get_related_class_function(base_class, classname, field):
```

Comment:
I can't seem to figure out what's changed – other than a bit of rearrangement this looks like the same code as before. Can you explain how this fixes the bug?

Reply:
I'll be honest with you:
I am not entirely sure why this works. Before the new monkey patched method was bound to the metaclass itself. Somehow this was affecting the model discovery, but to be fair, I came to this solution via a bit of trial and error.

It definitely makes a difference. If you want, you can undo my change and have a look at the error